src/main/resources/alfresco/repository.properties

```diff
@@ -834,12 +834,12 @@ encryption.reencryptor.numThreads=2
 
 # SOLR connection details (e.g. for JMX)
 solr.host=localhost
-solr.port=8080
-solr.port.ssl=8443
+solr.port=8983
+#solr.port.ssl=8983

```

Comment:
I would recommend using a different port in keeping with documentation.
It also looks weird in the ui to see 8983 for ssl and non ssl

Reply:
I could not find any mention of a different port for SSL and after talking to you my understanding was that it might be the same port. That's the reason why it is set to 8983 but commented out.

Reply:
It just look strange when you look at the ui and see that both are on the same port.

Reply:
Update the port number to 8984 (https://lucene.apache.org/solr/guide/6_6/enabling-ssl.html)